plugins/chat/assets/javascripts/discourse/components/chat/threads/index.gjs

```diff
@@ -0,0 +1,20 @@
+import Component from "@glimmer/component";
```

Comment:
This is an experimental solution which I would expand more after this PR, I want an all in one solution for every headers in chat and drawer too.